neural_speed/application/main_pybind.cpp

```diff
@@ -73,7 +73,8 @@ class Model {
   // deprecated API
   std::vector<std::vector<model_token>> generate_tokens(const std::vector<std::vector<model_token>>& input_ids);
   const std::vector<float>& evaluate_(const std::vector<std::vector<model_token>>& input_ids);
-  py::array_t<float> evaluate(const std::vector<std::vector<model_token>>& input_ids) {
+  py::array_t<float> evaluate(const std::vector<std::vector<model_token>>& input_ids, bool logits_all = false) {
+    if (logits_all) ctx->logits_all = true;
```

Comment:
Otherwise, it seems to be an unexpected behavior if called `evaluate` with `logits_all = true` and then `logits_all = false`
```suggestion
    ctx->logits_all = logits_all;
```

Reply:
thx